privacy-policy

```diff
@@ -0,0 +1,76 @@
+Nuts respecteert de privacy van de bezoekers van haar website.
+Nuts geeft de door u verstrekte persoonsgegevens niet door aan andere partijen, tenzij u hier nadrukkelijk toestemming voor hebt gegeven of hiertoe een verplichting bestaat op basis van wet- of regelgeving.
+
+## Begrippen
+Hieronder worden enkele definities gegeven van veel gebruikte termen in dit statement:
+
+Persoonsgegevens: elk gegeven over een natuurlijke persoon of gegeven dat herleidbaar is tot een bepaalde persoon zoals voornaam, achternaam, geboortedatum, ip-adres, locatiegegevens of adres.
+
+Verwerken van persoonsgegevens: elke handeling met betrekking tot het omgaan met persoonsgegevens, bijvoorbeeld verzamelen, vastleggen, verstrekken en vernietigen of archiveren.
+
+Cookies zijn kleine stukjes informatie (bestandjes) die een website op uw apparatuur (bijvoorbeeld uw computer) achterlaat. De website instrueert de webbrowser waarmee u websites bekijkt (bijvoorbeeld Internet Explorer of Google Chrome) om deze cookies op uw apparatuur op te slaan.
+
+IP-adres is een Internet Protocol adres dat verwijst naar een unieke bestemming op internet. Dit kan betekenen dat het IP-adres in bepaalde gevallen direct naar de computer van de bezoeker is te herleiden. Het IP-adres is essentieel om de pagina’s via internet naar de juiste computer te sturen.
+
+## Algemene Verordening Gegevensbescherming
+Nuts houdt zich aan de Nederlandse wet- en regelgeving omtrent de bescherming van persoonsgegevens en de Europese Privacy Verordening (EU/2016/679). Nuts behandelt de door u verstrekte persoonlijke gegevens met grootst mogelijke zorgvuldigheid en vertrouwelijkheid.
+
+## Communicatie
+Wanneer u een e-mail, een contactformulier op een van de websites of andere berichten naar Nuts verzendt, is het mogelijk dat die berichten als e-mail bewaard worden. Soms wordt naar uw persoonlijke gegevens, die voor de desbetreffende situatie relevant zijn, gevraagd. Dit maakt het mogelijk uw vragen te verwerken en uw verzoeken te beantwoorden. De gegevens worden opgeslagen op eigen beveiligde servers of die van een derde partij. 
```

Comment:
Moeten we hier nog even melden dat we mailchimp gebruiken? Aangezien we ook Google noemen als partner.